packages/volto/src/components/manage/Toolbar/PersonalTools.jsx

```diff
@@ -50,7 +47,9 @@ const PersonalTools = (props) => {
   const token = useSelector((state) => state.userSession.token, shallowEqual);
   const user = useSelector((state) => state.users.user);
   const userId = token ? jwtDecode(token).sub : '';
-
+  const siteSetupAction = find(props.actions?.user, {
```

Comment:
@Faakhir30 Can you remove lodash from here? We will likely get rid of it one day